src/util/aws-kms-utils.ts

```diff
@@ -21,26 +21,26 @@ const EcdsaPubKey = asn1.define("EcdsaPubKey", function (this: any) {
 /* eslint-enable func-names */
 
 export async function sign(digest: Buffer, kmsCredentials: AwsKmsSignerCredentials) {
-  const kms = new KMS(kmsCredentials);
-  const params: KMS.SignRequest = {
-    // key id or 'Alias/<alias>'
-    KeyId: kmsCredentials.keyId,
-    Message: digest,
-    // 'ECDSA_SHA_256' is the one compatible with ECC_SECG_P256K1.
-    SigningAlgorithm: "ECDSA_SHA_256",
-    MessageType: "DIGEST",
-  };
-  const res = await kms.sign(params).promise();
-  return res;
+  const kms = new KMSClient(kmsCredentials);
+  return kms.send(
+    new SignCommand({
+      // key id or 'Alias/<alias>'
+      KeyId: kmsCredentials.keyId,
+      Message: digest,
+      // 'ECDSA_SHA_256' is the one compatible with ECC_SECG_P256K1.
+      SigningAlgorithm: "ECDSA_SHA_256",
+      MessageType: "DIGEST",
+    })
+  );
 }
 
 export async function getPublicKey(kmsCredentials: AwsKmsSignerCredentials) {
```

Comment:
Do we still need `async` here? Not sure if the code still has any promises/await